pyproject.toml

```diff
@@ -194,6 +194,7 @@ exclude = [
 [tool.mypy]
 mypy_path = ["python"]
 files = ["python", "test/functional/python", "example/python_api"]
+exclude = ["_skbuild"]
```

Comment:
Don't need exclude because above it is inclusive list.

Reply:
Without it, I have the following
```
python/_skbuild/macosx-12.0-arm64-3.10/setuptools/lib.macosx-12.0-arm64-cpython-310/spglib/__init__.py: error: Duplicate module named "spglib" (also at "python/_skbuild/macosx-12.0-arm64-3.10/cmake-install/spglib/__init__.py")
python/_skbuild/macosx-12.0-arm64-3.10/setuptools/lib.macosx-12.0-arm64-cpython-310/spglib/__init__.py: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules for more info
python/_skbuild/macosx-12.0-arm64-3.10/setuptools/lib.macosx-12.0-arm64-cpython-310/spglib/__init__.py: note: Common resolutions include: a) using `--exclude` to avoid checking one of them, b) adding `__init__.py` somewhere, c) using `--explicit-package-bases` or adjusting MYPYPATH
```

Reply:
Oops, `_skbuild` seems to be created in some older version of spglib. I removed it.